src/main/webapp/script.js

```diff
@@ -114,17 +130,21 @@ function displaySentimentBucketChart(sentimentBucketList) {
  * top popular words and its appearance
  */
 function displayWordCloudChart(wordFrequencyMap) {
```

Comment:
If displaysentimentbucket can be created, wordcloud chart has no reason not to work. There might be some logic errors here

Reply:
I'm not sure what you mean. The way I have errors set up is so that if something breaks in any of the helper function, it'll throw a distinct error to help with debugging.

Reply:
sure if these two methods are called simultaneously. If the displaysentimentbucket is not working, the whole script.js should break and return to front end. So potentially there is no need to do try/catch for displaywordcloud frequency